content/blog/pointers-and-references-cpp/index.md

```diff
@@ -0,0 +1,153 @@
+﻿---
+{
+    title: 'Pointers and References in C/C++',
+    description: 'An overview of how pointers and references function in C/C++',
+    published: '2020-05-20T09:40:00.000Z',
+    edited: '2020-05-20T09:40:00.000Z',
+    authors: ['seanmiller'],
+    tags: ['computer science', 'cpp'],
+    attached: [],
+    license: 'cc-by-nc-nd-4'
+}
+---
+
+# Pointers and References
+
+Every new C/C++ programmer will eventually reach the point at which they are forced to work with pointers and will undoubtedly realize that they extremely dislike using them because they are a little complex. Today, we'll be looking at what pointers are, deconstructing their usage, and hopefully, making the usage of pointers easier to grok.
+
+  
+## What is a Pointer? {#what-is-a-pointer}
+
```

Comment:
Inconsistent white space, should be a non issue but will confirm that with the MD rendering engine. It shouldn't matter, but a note to myself of why I'm digging through packages

Reply:
Confirmed, will not be an issue. This is an issue in some MD parsers, but Remark should be just fine